src/preload/api/recording.api.ts

```
@@ -0,0 +1,28 @@
+import { ipcRenderer } from 'electron';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)